src/Administration/Resources/app/administration/src/module/sw-product-stream/page/sw-product-stream-detail/sw-product-stream-detail.html.twig

```diff
@@ -153,6 +153,29 @@
                     :class="{ 'is--disabled': !acl.can('product_stream.editor') }"
                     :title="$tc('sw-product-stream.detail.titleFilter')"
                 >
+                    <div
+                        v-if="!productStreamIndexingEnabled"
```

Comment:
Since there are no data computations in it, I would use `v-show` instead.

Reply:
Removing it from the DOM with v-if saves memory and DOM nodes if the user never sees it.
 I would use v-if here.